manifests/params.pp

```diff
@@ -38,7 +40,10 @@
   $time_servers = ['0.pool.ntp.org iburst', '1.pool.ntp.org iburst', '2.pool.ntp.org iburst', '3.pool.ntp.org']
 
   # where the agent installer tarball should go. This is only relevant when promoting a secondary master
-  $publicdir = '/opt/puppet/packages/public/classroom'
+  $publicdir = $aio ? {
```

Comment:
bah, that was a variable I calculated, but then realized that I could just use `aio_agent_version`